### ISSUE TITLE
Can't connect to Server

### ISSUE THREAD

**Issue by Krungs** [arkmanager/ark-server-tools#1229]
If I want to start my server I get following output:

Running command 'status' for instance 'Fjordur'
 Server running:   Yes
 Server PID:   1193
 Server listening:   Yes
Unable to query server
 Server online:   No
 Server build ID:   8907760
 Server version:   346.11

After 10 minutes it shows me that he is Online:

Running command 'status' for instance 'Fjordur'
 Server running:   Yes
 Server PID:   1193
 Server listening:   Yes
Server Name: SaltyWolves_Fjordur - (v346.11)
Steam Players: 0 / 10
Active Steam Players: 0
 Server online:   Yes
 ARKServers link:   http://arkservers.net/server/
 Steam connect link:   steam://connect/
 Server build ID:   8907760
 Server version:   346.11

 I can see the Server ingame but I cant connect. It is the only server running on this vm and I dont know what to do.

**Comment by klightspeed** [comment]
Did you kill `ARK #603298`?

I presume by "VM" you mean Docker container?

Are you binding the appropriate ports to the container, and forwarding those ports through your router?

If you specify `arkNoPortDecrement=true`, then you will need to bind UDP `ark_QueryPort` and TCP `ark_Port` and `ark_Port + 1`
If you do not specify that, you will need to bind UDP `ark_QueryPort` and TCP `ark_Port - 1` and `ark_Port`

**Comment by Krungs** [comment]
I have a root server on Hetzner with Proxmox as hypervisor and there I build a VM with Debian 10 to run the ark server(s).
I solved my problem yesterday night and it was like that what u say. I opened the ports on my firewall vm but for the `ark_Port`
was just 1 port open. so I opened the port -1 too and it worked. I forgot that ark needed this so thats the point why I was suprised. I had to reinstall the whole server a few months ago where everything was completely setted up and forgot to do this.
